test/integration/api.bats

```diff
@@ -207,9 +242,19 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker start" {
-	skip
+	start_docker 3 
+	swarm_manage
+	# create
+	run docker_swarm create --name test_container busybox sleep 1000
+	[ "$status" -eq 0 ]
+
```

Comment:
Can you verify if that's the case with docker as well?

You might have found another bug in swarm!